docs/content/querying/sql.md

```diff
@@ -149,7 +149,7 @@ String functions accept strings, and return a type appropriate to the function.
 
 |Function|Notes|
 |--------|-----|
-|`x \|\| y`|Concat strings x and y.|
+|`x &#124;&#124; y`|Concat strings x and y.|
```

Comment:
Hm, I don't think these escapes are working: https://github.com/apache/incubator-druid/blob/1d100a54f70c00729f0b33d32ae6a2a042b48cb7/docs/content/querying/sql.md

Reply:
After playing around a bit I found something that works:

```
|<code>x &#124;&#124; y</code>|Concat strings x and y.|
```

Reply:
It works! :+1:

Reply:
I found it works in https://github.com/zhaojiandong/druid/blob/master/docs/content/querying/sql.md
not work in https://github.com/apache/incubator-druid/blob/57d95fe194df9af00687203950fa86a2fe9e025d/docs/content/querying/sql.md